pkg/apis/policy/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,7 +1,7 @@
 //go:build !ignore_autogenerated
 
 /*
-Copyright  The cert-manager Authors.
+Copyright The cert-manager Authors.
```

Comment:
Why did this change?

Reply:
I changed the template so it does not include YEAR, and that caused the extra space to be removed.